sdk/programmableconnectivity/azure-programmableconnectivity/samples/LocationSample.py

```diff
@@ -0,0 +1,19 @@
+from azure.programmableconnectivity import ProgrammableConnectivityClient
```

Comment:
Please add a license header to each sample file:

```python
# ------------------------------------
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT License.
# ------------------------------------

```

Reply:
Also, in terms of the sample file names, we should use snake_case for Python

Reply:
changed both as suggested